lib/elixir/test/elixir/kernel/alias_test.exs

```diff
@@ -129,14 +137,25 @@ defmodule Macro.AliasTest.Aliaser do
   end
 end
 
-defmodule Macro.AliasTest.User do
+defmodule Macro.AliasTest do
```

Comment:
Is most likely no longer needs to be a separate module, we can fold into the previous one?